specs/trip_dispatch_spec.rb

```diff
@@ -89,4 +90,44 @@
       passenger.trips.must_include trip
     end
   end
+
+  describe "find_available_driver" do
+    it 'returns first available driver' do
+      @dispatcher = RideShare::TripDispatcher.new
+      driver = @dispatcher.find_available_driver
+      driver.must_be_instance_of(RideShare::Driver)
+      driver.status.must_equal(:AVAILABLE)
+      driver.name.must_equal("Emory Rosenbaum")
+    end
+  end
+
+  describe 'request_trip method' do
+    before do
+      @dispatcher = RideShare::TripDispatcher.new
```

Comment:
What if there is no available driver? This is an important case, and is exactly the sort of situation where software will do something unexpected. You might set up the test as follows:

```ruby
it 'raises an error if no driver is available' do
  @dispatcher.drivers.each do |driver|
    # This may require an attr_writer in Driver
    driver.status = :UNAVAILABLE
  end
  proc {
    @dispatcher.request_trip(2)
  }.must_raise StandardError
end
```